src/layout/LeftMenu/index.less

```diff
@@ -90,3 +90,10 @@ a:hover {
     }
   }
 }
+
+.bottom {
+  position: absolute;
+  bottom: 10px;
+  width: 50%;
```

Comment:
100% maybe is better?

Reply:
CSS has never been my favourite. 🥲

Changing width to 50% didn't make the text align very well, so I changed a few more settings. Please let me know if it still isn't right.